payments/vouchers.go

```diff
@@ -11,6 +11,16 @@ import (
 	"github.com/statechannels/go-nitro/types"
 )
 
+// ChannelId returns the channel id of the voucher
+func (v Voucher) ChannelId() types.Destination {
+	return v.channelId
+}
+
+// Amount returns the amount of the voucher
+func (v Voucher) Amount() *big.Int {
+	return v.amount
+}
+
```

Comment:
Is this a hint that these fields should just be exported / public?

Reply:
I think so! Especially since `Voucher` are exchanged in messages and on chain.

Reply:
I'm going to defer to a separate smaller PR.